commands/orchestrationcommands/stackeventcommands/common.go

```diff
@@ -0,0 +1,26 @@
+package stackeventcommands
+
+import (
+	"encoding/json"
+	"github.com/rackspace/rack/internal/github.com/fatih/structs"
+	osStackEvents "github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/openstack/orchestration/v1/stackevents"
+)
+
+func eventSingle(rawEvent interface{}) map[string]interface{} {
+	m := structs.Map(rawEvent)
+	switch event := rawEvent.(type) {
+	case *osStackEvents.Event:
+		if event.Time.Unix() != -62135596800 {
+			m["Time"] = event.Time
+		} else {
+			m["Time"] = "None"
```

Comment:
What I was trying to do here is to show "None" instead of "<nil>" when no information is provided. I think that was misguided since this is go and not python. Ill change that